src/Silakov.Homework5/Models/AnimalBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silakov.Homework5.Models
+{
+    public abstract class AnimalBase
+    {
+        /// <summary>
+        /// Название животного.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Возраст.
+        /// </summary>
+        public int Age { get; set; }
+
+        /// <summary>
+        /// Список характеристик.
+        /// </summary>
+        public List<string> listCharacteristics { get; set; }
```

Comment:
let's use a common naming convention for .net where properties start with uppercase character